nimx/composition.nim

```diff
@@ -207,7 +207,13 @@ vec4 composeDistanceFuncDebug(float dist) {
 }
 
 void drawShape(float dist, vec4 color) {
-    gl_FragColor = mix(gl_FragColor, color, fillAlpha(dist));
+    if (gl_FragColor.w == 0.0) {
```

Comment:
Could you please define another function e.g. `drawFirstShape` or smth? And then use it.